ouroboros-network-protocols/CHANGELOG.md

```diff
@@ -1,5 +1,19 @@
 # Revision history for ouroboros-network-protocols
 
+## Unreleased version
+
+### Breaking changes
+
+* Added kind signatures to protocol types: 
+  * `ChainSync`,
+  * `BlockFetch`,
+  * `KeepAlive`,
```

Comment:
I can not find the kind signature update for the KeepAlive protocol. Did you forget to commit it?

Reply:
`KeepAlive` doesn't have any polymorphic parameters, so there's no need.

Reply:
I updated the `CHANGELOG` file: it should mention `TxSubmission2` and not `KeepAlive`.